packages/medusa/src/api-v2/store/carts/[id]/route.ts

```diff
@@ -17,3 +20,23 @@ export const GET = async (req: MedusaRequest, res: MedusaResponse) => {
 
   res.json({ cart })
 }
+
+export const POST = async (req: MedusaRequest, res: MedusaResponse) => {
+  const updateCartWorkflow = updateCartsWorkflow(req.scope)
```

Comment:
suggestion: Normally you should be able to skip that step and pass the container in the run